libdnf/repo/repo.cpp

```diff
@@ -492,6 +496,11 @@ void Repo::internalize() {
 
 
 void Repo::reset_metadata_expired() {
```

Comment:
`reset_metadata_expired` does not have a doc string. I know it is a private method but anyway it would be nice to have it because the name is somehow confusing to me.